Update homepage subject to review

arnoldnekemiah opened the pull request:
- I have updated MIT.md which is from the template provided.
- I have added index.html file
- I have added style.css file

I am submitting for review

arnoldnekemiah committed:
add .gitignore file

arnoldnekemiah committed:
Update .ignore file

arnoldnekemiah committed:
Create linters file

arnoldnekemiah committed:
Add index.html file

arnoldnekemiah committed:
Add style.css file

arnoldnekemiah committed:
Added MIT.md to .ignore file

arnoldnekemiah committed:
Update Readme.md file

Ibizugbe commented on .gitignore:
```diff
@@ -0,0 +1,3 @@
+node_modules
+
+MIT.md 
```
It is a[ requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) that you create a test.md file with some text that should be added to your `.`gitignore` file😉I think you haven't implemented that currently😉😉 kindly fix that🤗
also, we are not required to ignore the `MIT` file as this is the license file for this project😉 I suggest you kindly take it away🤗🤗

<img width="572" alt="test_imp2" src="https://user-images.githubusercontent.com/64643995/218956700-e094209e-b693-4a90-baad-c2bdfd6d8b1e.png">

Ibizugbe reviewed (changes requested):
Hi @arnoldnekemiah 👋👋, 

![198009982-d99c2507-f20d-4c00-90a3-a54097598b36](https://user-images.githubusercontent.com/64643995/218955336-531f92f0-5ffd-4196-a67d-60446903317f.gif)


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights👍👍

- [x] Github flow has been used👍


## Required Changes ♻️

- [ ] It is a requirement in this milestone that linters are properly setup, which you have already done👍 but it is also a requirement that all linter tests should pass and currently, your stylelint test is failing😉 kindly fix this issue🤗
<img width="743" alt="faiked__stylelint" src="https://user-images.githubusercontent.com/64643995/218955854-9e1e343b-5357-4db7-bbb4-231633c0d120.png">

- [ ] Nice job building up a readme file for this project👍 but currently, I think the contents of your readme file are not related to this milestone😉 I suggest you write a descriptive readme file, with MIT license for this milestone, It should make the user understand what your project is about and how to use it😉 kindly make use of the microverse template and ensure every section is filled🤗 don't for get it is a [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) that our project is professionally documented😉😉

I suggest you go through the [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) for this project again and use it as a guide to make the necessary change😉🤗

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

arnoldnekemiah committed:
Update .gitignore file

Remove MIT.md as suggested by the code reviewer

arnoldnekemiah committed:
Update stylelint

Update style lint to run dependencies and update

arnoldnekemiah committed:
Update linters

Tried copying linter.yml code once more from the documentation offered by microverse

arnoldnekemiah committed:
Update style.css

arnoldnekemiah committed:
comment out code for style.css content

To check viability of linters

arnoldnekemiah committed:
Update style.css

arnoldnekemiah committed:
remove css content
